-------------------------------------------------------------------------------
Test set: TestSuite
-------------------------------------------------------------------------------
Tests run: 20, Failures: 8, Errors: 0, Skipped: 0, Time elapsed: 67.313 s <<< FAILURE! - in TestSuite
test1(com.RetryLogic.RetryLogicTest)  Time elapsed: 0.007 s  <<< FAILURE!
java.lang.AssertionError: expected [false] but found [true]
	at com.RetryLogic.RetryLogicTest.test1(RetryLogicTest.java:11)

test2(com.RetryLogic.RetryLogicTest)  Time elapsed: 0 s  <<< FAILURE!
java.lang.AssertionError: expected [false] but found [true]
	at com.RetryLogic.RetryLogicTest.test2(RetryLogicTest.java:16)

testDataMethod(com.dataProvider.test.dataprovider)  Time elapsed: 2.29 s  <<< FAILURE!
java.lang.NullPointerException
	at com.dataProvider.test.dataprovider.testDataMethod(dataprovider.java:44)

testDataMethod(com.dataProvider.test.dataprovider)  Time elapsed: 2.319 s  <<< FAILURE!
java.lang.NullPointerException
	at com.dataProvider.test.dataprovider.testDataMethod(dataprovider.java:44)

testDataMethod(com.dataProvider.test.dataprovider)  Time elapsed: 1.091 s  <<< FAILURE!
java.lang.NullPointerException
	at com.dataProvider.test.dataprovider.testDataMethod(dataprovider.java:44)

testDataMethod(com.dataProvider.test.dataprovider)  Time elapsed: 1.282 s  <<< FAILURE!
java.lang.NullPointerException
	at com.dataProvider.test.dataprovider.testDataMethod(dataprovider.java:44)

testDataMethod(com.dataProvider.test.dataprovider)  Time elapsed: 1.421 s  <<< FAILURE!
java.lang.NullPointerException
	at com.dataProvider.test.dataprovider.testDataMethod(dataprovider.java:44)

testDataMethod(com.dataProvider.test.dataprovider)  Time elapsed: 1.332 s  <<< FAILURE!
java.lang.NullPointerException
	at com.dataProvider.test.dataprovider.testDataMethod(dataprovider.java:44)

